boards/adafruit-feather-rp2040/examples/adafruit_feather_neopixel_rainbow.rs

```diff
@@ -52,7 +51,7 @@ fn main() -> ! {
     );
 
     let timer = Timer::new(pac.TIMER, &mut pac.RESETS, &clocks);
-    let mut delay = timer.count_down();
+    let mut delay = timer;
```

Comment:
Could the timer be called delay from the start or the uses of delay be changed to timer instead?

Reply:
Did you mean like this https://github.com/rp-rs/rp-hal-boards/pull/60/commits/c42bbdf949cbd59a6b3a3c86958b7e2f37591a5d ?

Reply:
Yes exactly :)
You can keep all these changes in the same commit if you don't feel like doing revising :)
I would've done all the (similar) changes in a single commit rather than 1 per board because I'm lazy xD

Reply:
Yes, I'll do all of them in the same commit, I just wanted to make sure I understood you correctly and changed just one example at the moment :)

Thanks for the feedback.